packages/generator/src/artifacts/types/main.ts

```diff
@@ -14,7 +14,7 @@ export const generateTypesEntry = () => ({
       export function defineGlobalStyles(definition: GlobalStyleObject): GlobalStyleObject
       export function defineTextStyles(definition: CompositionStyles['textStyles']): CompositionStyles['textStyles']
       export function defineLayerStyles(definition: CompositionStyles['layerStyles']): CompositionStyles['layerStyles']
-      export function definePattern<T>(config: PatternConfig<T>): PatternConfig
+      export function definePattern<T extends PatternConfig>(config: PatternConfig<T>): PatternConfig
```

Comment:
I think it should be  `  export function definePattern<T extends PatternProperties>(config: PatternConfig<T>): PatternConfig
` instead.